msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md

```diff
@@ -51,7 +51,7 @@ In the **Configure** section, you can see the following components to manage and
   * Meeting extension
   * Activity feed notification
 * **Permissions**: This section allows you to give Device permissions, Team permissions, Chat or Meeting permissions, and User Permissions for your app.
-* **Single sign-on**: You can configure your app to authenticate users with single sign-on (SSO).
+* **Single sign-on**: Developer Portal supports the Single sign-on (SSO) configured for the Teams app in Azure AD. Add the **Application ID URI** that you configured for your app in the Azure AD.
```

Comment:
this is just the manifest field for SSO. We should be mentioning that Bot Registered on Azure AD supports Single Sign on. If a bot is registered on Bot Framework Portal (or within the developer Portal under Bot Management) then those bots don't support SSO and have to be Registered on Azure for them to Support SSO.

For a bot registered on Azure, add the Application ID URI, Refer to [this link](https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/authentication/auth-aad-sso-bots) on how to get Application URI

Reply:
Developer Portal supports the Single sign-on (SSO) : Please remove this line as developer portal does not support SSO, it only ask for the user to input the Application URI for Bots provisioned on Azure